packages/contracts-bedrock/book/src/policies/versioning.md

```diff
@@ -107,15 +107,18 @@ discouraged. If a change is required to a release candidate after it has been ta
 
 ### Additional Release Candidates
 
-Sometimes additional release candidate versions are needed, in that case, the follow process should be used.
-This process is designed to (1) ensures fixes are made on both the release and the trunk branch
- and (2) avoids the need to stop development efforts on the trunk branch.
+Sometimes fixes or additional changes need to be added to a release candidate version. In that case
+we want to ensure fixes are made on both the release and the trunk branch, without stopping development
+efforts on the trunk branch.
 
+The process is as follows:
 
-1. Make the fixes on `develop`. *For whatever the normal semver level increment should be, bump that value by 5.*
+1. Make the fixes on `develop`. Increment the contracts semver as normal.
 2. Create a new release branch, named `proposal/op-contracts/X.Y.Z-rc.n+1` off of the rc tag.
```

Comment:
This is not a great name for the branch. It implies create a new release branch for every patch which is unnecessary.  Only one release branch should be created per release. So if we have `op-contracts/v4.0.0-rc.2` tagged off develop and need to add a fix we should create `proposal/op-contracts/4.0.0` branch off of the `op-contracts/v4.0.0-rc.2` tag.  We cherry-pick the fix to that branch and then tag `op-contracts/v4.0.0-rc.3` off the branch.

If later another fix is required for the release, we add it to the existing `proposal/op-contracts/4.0.0` branch and tag a new RC.

Reply:
If I understand correctly, you are suggesting this change which makes sense to me. 

```suggestion
2. Create a new release branch, named `proposal/op-contracts/X.Y.Z` off of the rc tag.
```

Reply:
That flow from @ajsutton makes sense, and the new suggestion seems right to me